Sources/ApolloCodegenLib/ApolloCodegenOptions.swift

```diff
@@ -86,7 +86,7 @@ public struct ApolloCodegenOptions {
   ///  - outputFormat: The `OutputFormat` enum option to use to output generated code.
   ///  - customScalarFormat: How to handle properties using a custom scalar from the schema.
   ///  - suppressSwiftMultilineStringLiterals: Don't use multi-line string literals when generating code. Defaults to false.
-  ///  - urlToSchemaFile: The URL to your schema file.
+  ///  - urlToSchemaFile: The URL to your schema file. Accepted file types are `.json` for JSON files, or either `.graphqls` or `.sdl` for Schema Definition Language files.
```

Comment:
Is `.sdl` valid, will ApolloCLI recognize that? This will likely change in 1.0 though as the frontend looks specifically for .json otherwise attempts to interpret any file as SDL - [here](https://github.com/apollographql/apollo-ios/blob/release/1.0-alpha-incubating/Sources/ApolloCodegenLib/Frontend/GraphQLJSFrontend.swift#L44).

Reply:
There are still recommended default extensions though, like [this](https://github.com/apollographql/apollo-ios/blob/release/1.0-alpha-incubating/Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift#L198).